databases/week1/databases_week1.sql

```diff
@@ -0,0 +1,64 @@
+-- Databases lesson 1 Homework
+USE hyf_class2023;
+
+-- 1. Find out how many tasks are in the task table
+SELECT COUNT(id) AS "TOTAL TASKS"
+FROM task;
+
+-- 2. Find out how many tasks in the task table do not have a valid due date
+SELECT count(title) AS "Task Without Valid Due Date"
+FROM task
+WHERE due_date IS NULL;
+
+-- 3. Find all the tasks that are marked as done
+SELECT title As "Tasks", name As "Status"
+FROM task
+INNER JOIN status
+ON task.status_id = status.id
+WHERE name = "Done";
```

Comment:
it is a good habit to use table aliases for all fields when there is more than one table. 
This makes the code more readable and helps to prevent logical errors. Also in the case when both tables have the same field name, for example, `id`, `name`, `created_date`, etc.


```suggestion
SELECT task.title As "Tasks", status.name As "Status"
FROM task
INNER JOIN status
ON task.status_id = status.id
WHERE status.name = "Done";
```

Reply:
Made necessary changes.